src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -50,27 +53,45 @@ public class FileService implements DisposableBean {
 
     private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(Runtime.getRuntime().availableProcessors());
 
+    // NOTE: this list has to be the same as in file-uploader.service.ts
+    private final List<String> allowedFileExtensions = new ArrayList<>(Arrays.asList("png", "jpg", "jpeg", "svg", "pdf", "zip"));
```

Comment:
We should make sure that this does not create functionality conflicts with #5578 where these file extensions are extended and their handling is changed. Just as a heads up

Reply:
I already have this on my radar. Waiting for it to be merged first.